server/api/src/test/java/org/hawkular/apm/server/api/model/zipkin/SpanTest.java

```diff
@@ -114,8 +114,7 @@ public void testSerialization() throws IOException, ClassNotFoundException {
         binaryAnnotationWithMapping.setEndpoint(createEndpoint("bonjour2", "127.0.1.2", 8090));
         binaryAnnotationWithMapping.setType(AnnotationType.DOUBLE);
 
-        Span span = new Span(Arrays.asList(stringAnnotation, binaryAnnotationWithMapping));
-        span.setAnnotations(serverAnnotations());
+        Span span = new Span(Arrays.asList(stringAnnotation, binaryAnnotationWithMapping), serverAnnotations());
 
         Span deserializedSpan = (Span)deserialize(serialize(span));
```

Comment:
Could you add unit tests for clientSpan and serverSpan methods, now that the implementation has changed.